### ISSUE TITLE
[Bug] Code Loader Check Failing

### ISSUE THREAD

**Issue by dbelleman** [thesupersonic16/HedgeModManager#110]
Updated earlier today and just re-downloaded still same issue. Worked great yesterday before update.

HMM Info:
```
    Version: 7.2-4
    Args: 
    StartDir: C:\Program Files (x86)\Steam\steamapps\common\Sonic Generations\
    Process Level: User
    Game: Sonic Generations
    SteamGame: Sonic Generations
```

Exception:
```
    Type: ArgumentNullException
    Message: Value cannot be null.
Parameter name: input
    Source: mscorlib
    Function: System.Version Parse(System.String)
    StackTrace: 
       at System.Version.Parse(String input)
       at System.Version..ctor(String version)
       at HedgeModManager.App.GetCodeLoaderInfo(Game game)
       at HedgeModManager.MainWindow.CheckCodeCompatibility()
       at HedgeModManager.MainWindow.RefreshMods()
       at HedgeModManager.MainWindow.Window_Loaded(Object sender, RoutedEventArgs e)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.BroadcastEventHelper.BroadcastEvent(DependencyObject root, RoutedEvent routedEvent)
       at System.Windows.BroadcastEventHelper.BroadcastLoadedEvent(Object root)
       at MS.Internal.LoadedOrUnloadedOperation.DoWork()
       at System.Windows.Media.MediaContext.FireLoadedPendingCallbacks()
       at System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
       at System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
       at System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
       at System.Windows.Interop.HwndTarget.OnResize()
       at System.Windows.Interop.HwndTarget.HandleMessage(WindowMessage msg, IntPtr wparam, IntPtr lparam)
       at System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
       at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
       at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
       at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
       at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.UnsafeNativeMethods.CallWindowProc(IntPtr wndProc, IntPtr hWnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.HwndSubclass.DefWndProcWrapper(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.UnsafeNativeMethods.CallWindowProc(IntPtr wndProc, IntPtr hWnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
    InnerException: 
```

**Comment by thesupersonic16** [comment]
Can you please check if this solves your issue?
[HedgeModManager.zip](https://github.com/thesupersonic16/HedgeModManager/files/4777819/HedgeModManager.zip)

**Comment by dbelleman** [comment]
Fixed it. Just a heads up. The Infinite Boost Gauge code crashes the game before it can even load. Worked before update.

**Comment by thesupersonic16** [comment]
I'll have the code checked. But did the new exe solve the issue for you?

**Comment by dbelleman** [comment]
> 
> 
> I'll have the code checked. But did the new exe solve the issue for you?

Like a charm. Great work.

**Comment by Sajidur78** [comment]
> Just a heads up. The Infinite Boost Gauge code crashes the game before it can even load.

The code has been fixed on b713c707f88f45a37d0b2e430cca1af4e06f593a.
Downloading community codes should grab the fix.
